### ISSUE TITLE
ls.mumps does not work with multiple right-hand sides

### ISSUE THREAD

**Issue by rc** [sfepy/sfepy#1023]
Demonstration:
```
import numpy as np
import scipy.sparse as sps

from sfepy.base.base import Struct
from sfepy.solvers import Solver

ls = Solver.any_from_conf(Struct(name='ls', kind='ls.mumps'))

num = 5
nc = 2

rhs = np.cumsum(np.ones((num, nc), dtype=np.float64), axis=1)

# mtx = sps.eye(num, dtype=np.float64)
mtx = sps.csr_matrix(
    np.array([[3, 1, 0, 0, 0],
              [1, 3, 1, 0, 0],
              [0, 1, 3, 1, 0],
              [0, 0, 1, 3, 1],
              [0, 0, 0, 1, 3],]), dtype=np.float64)

sol1 = ls(rhs, mtx=mtx)
val1 = sol1.sum(axis=0)

sol2 = []
for ic in range(nc):
    sol2.append(ls(rhs[:, ic], mtx=mtx))
sol2 = np.array(sol2).T
val2 = sol2.sum(axis=0)

print(rhs)

print(sol1)
print(sol2)

print(val1)
print(val2)
```
leads to
```
[[1. 2.]
 [1. 2.]
 [1. 2.]
 [1. 2.]
 [1. 2.]]
[[ 0.11111111  0.66666667]
 [-0.11111111  0.66666667]
 [ 0.11111111  2.        ]
 [ 1.          2.        ]
 [ 1.          2.        ]]
[[0.27777778 0.55555556]
 [0.16666667 0.33333333]
 [0.22222222 0.44444444]
 [0.16666667 0.33333333]
 [0.27777778 0.55555556]]
[2.11111111 7.33333333]
[1.11111111 2.22222222]
```
and no error is reported .

**Comment by rc** [comment]
Tested with libmumps-5.2.1, libmumps-5.4.

**Comment by vlukes** [comment]
The multiple right hand side vectors have not yet been implemented. Fixed by PR #1024.

**Comment by rc** [comment]
Great, thanks!
